uobtheatre/productions/migrations/0025_auto_20220903_1859.py

```diff
@@ -0,0 +1,41 @@
+# Generated by Django 3.2.15 on 2022-09-03 18:59
+
+import django.core.validators
+import django.db.models.deletion
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("productions", "0024_merge_20220808_2050"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="production",
+            name="excerpt_text",
+            field=models.CharField(blank=True, max_length=255, null=True),
+        ),
+        migrations.AlterField(
```

Comment:
There are a few migrations here that shouldn't be in this pr. This isnt your fault but we should fix this elsewhere so let's not merge this until weve added the missing migrations to dev. Once we've done that you will have to remove this migration and regenerate.